utils_cv/detection/model.py

```diff
@@ -236,6 +282,9 @@ def fit(
     ) -> None:
         """ The main training loop. """
 
+        if not self.dataset:
+            return
```

Comment:
Should this throw an error rather than returning silently. At the very least this needs a warning message, but throwing an error seems preferred.

Reply:
Done.